test/ctia/bundle/routes_test.clj

```diff
@@ -505,7 +528,20 @@
 
        (is (= 2 (count (:sightings bundle-get-res-4))))
        (is (nil? (:indicators bundle-get-res-4)))
-       (is (= 402 (count (:relationships bundle-get-res-4))))))))
+       (is (= 402 (count (:relationships bundle-get-res-4))))
+
+       (is (= (count (:sightings bundle-get-res-3))
+              (count (:sightings bundle-get-res-5))))
+       (is (= (count (:relationships bundle-get-res-3))
+              (count (:relationships bundle-get-res-5))))
+       (is (= (count (:indicators bundle-get-res-3))
+              (count (:indicators bundle-get-res-5))))
+
+       (is (empty? (:relationships bundle-from-source)))
+
+       (is (every? #(contains? #{sighting-id-1 sighting-id-2}
+                               (:target_ref %))
+                   (:relationships bundle-from-target)))))))
```

Comment:
Same here, I would only export with one Sighting ID to check that only the related entities are returned.